### DISCUSSION TITLE
how to see prompt of every available style?

### DISCUSSION THREAD

**Discussion by andreafebrian** [lllyasviel/Fooocus#1515] [Q&A] [answered]
hi guys i wondering how to see the prompt from the fooocus style tab? so i can use it on automatic1111?

**Comment by Hypersonic-Popcorn** [comment, marked as answer]
Inside of your where ever you installed \Fooocus\Fooocus\sdxl_styles directory there are a few json files. You'll find the prompts inside of those.

**Reply by andreafebrian** [reply]
thank you mate, really appreciate your answer.

**Reply by K3nt3L** [reply]
With the Ruined fork you've got a "send style to prompt" button. The easy way to read style directly into Foocus.
[RuinedFoocus](https://github.com/runew0lf/RuinedFooocus)

**Comment by xhoxye** [comment]
The expansion in the A1111 has some styles

**Reply by andreafebrian** [reply]
can you explain more?

**Reply by xhoxye** [reply]
https://github.com/ahgsql/StyleSelectorXL
![QQ截图20231220210639](https://github.com/lllyasviel/Fooocus/assets/129571231/e9ea47c9-cb83-49b4-b8ae-a437ce73027b)
